_config.yml

```diff
@@ -11,7 +11,6 @@ incremental: true
 permalink: pretty
 safe: false
 lsi: false
-url: https://docs.docker.com
```

Comment:
We need to double-check if this has no side-effects. I couldn't find any, but perhaps may have missed some